content/en/about/community/toc/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Technical Oversight Committee
+description: Information describing the Istio technical oversight committee.
+weight: 12
+keywords: [community]
+icon: community
+---
+Istio's Technical Oversight Community's [charter](https://github.com/istio/community/blob/master/TECH-OVERSIGHT-COMMITTEE.md#charter) and [membership](https://github.com/istio/community/blob/master/TECH-OVERSIGHT-COMMITTEE.md#committee-members).
```

Comment:
Community vs Committee?  We seem to have multiple instances of both.

Reply:
ya, this should be fixed :).